static/app/views/dashboards/datasetConfig/issues.tsx

```diff
@@ -78,7 +78,9 @@ function getTableSortOptions(organization: Organization, _widgetQuery: WidgetQue
   );
   const sortOptions = [...ISSUE_WIDGET_SORT_OPTIONS];
   if (hasBetterPrioritySort) {
-    sortOptions.push(IssueSortOptions.BETTER_PRIORITY);
+    sortOptions.unshift(IssueSortOptions.BETTER_PRIORITY);
+    const index = sortOptions.indexOf(IssueSortOptions.PRIORITY);
+    sortOptions.splice(index, 1);
```

Comment:
<img width="764" alt="Screenshot 2023-06-07 at 5 22 36 PM" src="https://github.com/getsentry/sentry/assets/29959063/a137012a-a921-45ce-912a-f4ba7e79b150">

This adds it to the list and removes the existing priority, but I can't find where to make it the default, that's still last seen.